README.md

```diff
@@ -1,24 +1,23 @@
+[![Bonsai Asset Badge](https://img.shields.io/badge/Sensu%20Slack%20Handler-Download%20Me-brightgreen.svg?colorB=89C967&logo=sensu)](https://bonsai.sensu.io/assets/sensu/sensu-slack-handler) [![Build Status](https://travis-ci.org/sensu/sensu-slack-handler.svg?branch=master)](https://travis-ci.org/sensu/sensu-slack-handler)
+
 # Sensu Go Slack Handler
 
 The Sensu Slack handler is a [Sensu Event Handler][1] that sends event data to
 a configured Slack channel.
 
-## Installation
+## Configuration
 
-Download the latest version of the sensu-slack-handler from [releases][2],
-or create an executable script from this source.
+### Asset registration
 
-From the local path of the slack-handler repository:
-```
-go build -o /usr/local/bin/sensu-slack-handler main.go
-```
+Assets are the best way to make use of this handler. If you're not using an asset, please consider doing so! If you're using sensuctl 5.13 or later, you can use the following command to add the asset: 
 
-## Configuration
+`sensuctl asset add sensu/sensu-slack-handler:1.0.3`
```

Comment:
I know I gave an example with the version included, but I think leaving it out will keep things low-touch and avoid the perception that the readme is out of date.

```suggestion
`sensuctl asset add sensu/sensu-slack-handler`
```